src/seedu/addressbook/AddressBook.java

```diff
@@ -18,43 +18,39 @@
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.Collections;
+import java.util.HashMap;
 import java.util.HashSet;
 import java.util.Optional;
 import java.util.Scanner;
 import java.util.Set;
 
 /*
- * NOTE : =============================================================
- * This class header comment below is brief because details of how to
- * use this class are documented elsewhere.
+ * NOTE : ============================================================= This
```

Comment:
errr! i guess this is <kbd>Ctrl</kbd> <kbd>Shift</kbd> <kbd> f </kbd> effect?
need to use it judiciously :-)